app.rb

```diff
@@ -0,0 +1,137 @@
+require_relative 'book'
+require_relative 'classroom'
+require_relative 'person'
+require_relative 'rental'
+require_relative 'student'
```

Comment:
![image](https://github.com/noelfoka/library-school/assets/93039359/8494f3ce-8b8c-42c2-a4a0-ac5f45cb871a)
![image](https://github.com/noelfoka/library-school/assets/93039359/05ceef29-d958-49a9-9010-962f7ee4c18b)

- [ ] Again it would be great if you replace the colon with the equal sign at both places in the constructor to fix the error of your app 👍

Reply:
This change has been done well